src/compiler/Stage/Parse/Parser.elm

```diff
@@ -410,23 +410,74 @@ literalInt =
         |> P.inContext InLiteralInt
 
 
-{-| TODO escapes
-TODO Unicode escapes
--}
+-- for literalChar and, in the future, literalString
+stringHelp = 
+    P.oneOf
+    [ P.succeed (identity)
```

Comment:
probably unneeded parens for the identity

Reply:
Fixed by https://github.com/elm-in-elm/compiler/pull/20/commits/657fa28321055d477fe637fcf45d8212664cefc6